libminifi/include/processors/ExtractText.h

```diff
@@ -50,15 +50,26 @@ class ExtractText : public core::Processor {
     //! Supported Properties
     static core::Property Attribute;
     static core::Property SizeLimit;
+
+    static core::Property RegexMode;
+    static core::Property IgnoreCaptureGroupZero;
+    static core::Property InsensitiveMatch;
+    static core::Property MaxCaptureGroupLen;
+    static core::Property EnableRepeatingCaptureGroup;
+
     //! Supported Relationships
     static core::Relationship Success;
     //! Default maximum bytes to read into an attribute
     static constexpr int DEFAULT_SIZE_LIMIT = 2 * 1024 * 1024;
 
     //! OnTrigger method, implemented by NiFi ExtractText
-    void onTrigger(core::ProcessContext *context, core::ProcessSession *session) override;
+    void onTrigger(core::ProcessContext *context, core::ProcessSession *session);
     //! Initialize, over write by NiFi ExtractText
-    void initialize(void) override;
```

Comment:
@phrocker : just found that I accidentally left these here when I fixed https://issues.apache.org/jira/browse/MINIFICPP-684 . It's part of core, It was released in scope of 0.6.0 and it seems we received no complains. Do you think this fact can be used to justify https://issues.apache.org/jira/browse/MINIFICPP-29 ? I'm happy to do further verification in this area, this just made me guess that override should be fine.